src/main/webapp/sass/_core.scss

```
@@ -422,6 +422,12 @@ end sticky footer
     color: $synapse-dark-text;
   }
 
+  @media (max-width: 500px) {
```

768px is the common convention in the codebase for narrow screen widths
```suggestion
  @media (max-width: 768px) {
```